prow/cluster/jobs/istio/bots/istio.bots.master.gen.yaml

```diff
@@ -136,6 +139,7 @@ presubmits:
           privileged: true
       nodeSelector:
         testing: test-pool
+    trigger: '(?m)^/test (?:.*? )?lint(?: .*?)?$'
```

Comment:
We should delete this and rerun_command. The command for retesting the `lint_bots` context should be straightforward: `/test lint_bots`. This is the default behavior.

@howardjohn @clarketm